DiscordBot/Models/Signup.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace DiscordBot.Models
+{
+    class Signup
+    {
+        public string Id;
+        public string Role;
```

Comment:
Any reason not to add getters only for these and to set them through a constructor? 🙂